client/src/screens/Product.js

```diff
@@ -80,16 +80,52 @@ export default ({ navigation }) => {
       <Carousel
         data={product.images}
         renderItem={carouselImages}
-        itemWidth={650}
-        sliderWidth={650}
+        itemWidth={250}
+        sliderWidth={350}
```

Comment:
```suggestion
        itemWidth="250px"
        sliderWidth="350px"
```
Just for clarity.

Reply:
it seems spec of Carousel??
"px" did not work...